app/assets/stylesheets/modules/_tagged_courses.styl

```diff
@@ -0,0 +1,15 @@
+.tag-download-csv 
```

Comment:
Can existing classes be used to style this button instead? This much styling for a single-use CSS class is a code smell.

Reply:
![Screenshot from 2024-10-10 11-16-35](https://github.com/user-attachments/assets/9c8115e9-2eb3-420d-8562-4a1bb6bba70b)

Will this work? It's using the existing CSS.